python/distrdf/spark/test_reducer_merge.py

```diff
@@ -302,6 +302,21 @@ def test_distributed_snapshot_columnlist(self):
         for filename in tmp_files:
             os.remove(filename)
 
+    def test_redefine_one_column(self):
+        """Test that values of one column can be properly redefined."""
+        # A simple dataframe with ten sequential numbers from 0 to 9
+        df = Spark.RDataFrame(10, sparkcontext=self.sc)
+        df_before = df.Define("x", "1")
```

Comment:
Might be good to check if we can have tests that are generic and can run for all backends, like this we would avoid the repetition (in this case Spark and Dask).

Reply:
The duplication is there to test that actually both Spark and Dask can run the same test. I think making generic tests could be done, we would need a common, mock backend only for such tests. We can discuss if this is needed and draft another PR